crates/core_simd/src/vectors_u8.rs

```diff
@@ -2,7 +2,7 @@
 
 /// A SIMD vector of containing `LANES` `u8` values.
 #[repr(simd)]
-pub struct SimdU8<const LANES: usize>([u8; LANES]);
+pub struct SimdU8<const LANES: usize>([u8; LANES]) where Self: crate::LanesAtMost64;
```

Comment:
Whoops, this is a mistake on my part.  I was too excited to publish the PR... So good catch because this is critical 🙃 This bound is completely unnecessary, but this is the one that prevents using the type in a way that causes a monomorpisation error.